### PR TITLE
record ReduceCPULimits result err info if err returned

### PR DESCRIPTION
**Pull request by NickrenREN**
record ReduceCPULimits result err info if err returned for debug

### CODE REVIEW

**Comment by NickrenREN** [comment]
@dims  @feiskyer  Could one of you help ask k8s-bot to test? Thanks

**Comment by NickrenREN** [comment]
keep-open  and /cc @dchen1107 @vishh   Do have have time? PTAL ,thanks

**Comment by NickrenREN** [comment]
@k8s-bot ok to test

**Comment by vishh** [comment]
Sorry, we are severely limited on review bandwidth. If your PR is fixing an
issue, please make it clear in the PR title. If it is just a refactoring,
it will have to wait.

**Comment by NickrenREN** [comment]
@vishh  Thanks.I cannot add labels. Could you help add lgtm and release-note labels .

File: pkg/kubelet/kubelet_pods.go [hunk 1]
```diff
@@ -572,7 +572,9 @@ func (kl *Kubelet) killPod(pod *v1.Pod, runningPod *kubecontainer.Pod, status *k
 	// and defer the responsibilty of destroying the pod's cgroup to the
 	// cleanup method and the housekeeping loop.
 	if reduceCpuLimts {
-		pcm.ReduceCPULimits(podCgroup)
+		if err := pcm.ReduceCPULimits(podCgroup); err != nil {
+			glog.Warningf("Reduces the CPU values to the minimum amount of shares error : %v", err)
```
Comment (dchen1107) [inline pkg/kubelet/kubelet_pods.go]: Changed the logging message to: Failed to reduce the CPU limit to ...?

**Comment by NickrenREN** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[pkg/kubelet/kubelet_pods.go, line 576 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39179#-KZdOKVVtI_zacov8LVT-r1-576:-Kc8EMmEaXKJqMeL0qYd:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/395abec616031da7e5953b3f71a0fafc5df81751/pkg/kubelet/kubelet_pods.go#L576)):*
<details><summary><i>Previously, dchen1107 (Dawn Chen) wrote…</i></summary><blockquote>

Changed the logging message to: Failed to reduce the CPU limit to ...?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39179)*

File: pkg/kubelet/kubelet_pods.go (hunk 1 above)
Comment (NickrenREN) [inline pkg/kubelet/kubelet_pods.go]: @dchen1107  Modified. PTAL

**Comment by NickrenREN** [comment]
@dchen1107  PTAL. Thanks

**Comment by NickrenREN** [comment]
@timstclair  Thanks. Could you please help add release-note-note label?

**Pull request merged**